.github/workflows/ci.yml

```
@@ -53,3 +53,39 @@ jobs:
       env:
         DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Check for generated files
+      if: github.repository == 'tldr-pages/tldr' && github.ref == 'refs/heads/main'
+      id: check-files
+      run: |
+        echo "zip_exists=$(if [[ $(find language_archives -name '*.zip' | wc -l) -gt 0 ]]; then echo true; else echo false; fi)" >> $GITHUB_ENV
+        echo "pdf_exists=$(if [[ $(find scripts/pdf -name '*.pdf' | wc -l) -gt 0 ]]; then echo true; else echo false; fi)" >> $GITHUB_ENV
+        echo "checksums_exists=$(if [[ -f 'tldr.sha256sums' ]]; then echo true; else echo false; fi)" >> $GITHUB_ENV
```

Using `-n` to check if `find` found something eliminates the need for `wc`.
`-quit` makes `find` stop searching after the first match, because we don't need the rest anyway.

I also added some newlines, because I think it's hard to read on one line.
```suggestion
if [[ -n $(find language_archives -name "*.zip" -print -quit) ]]; then
    echo "zip_exists=true" >> $GITHUB_ENV
else
    echo "zip_exists=false" >> $GITHUB_ENV
fi

if [[ -n $(find scripts/pdf -name "*.pdf" -print -quit) ]]; then
    echo "pdf_exists=true" >> $GITHUB_ENV
else
    echo "pdf_exists=false" >> $GITHUB_ENV
fi

if [[ -f tldr.sha256sums ]]; then
    echo "checksums_exist=true" >> $GITHUB_ENV
else
    echo "checksums_exist=false" >> $GITHUB_ENV
fi
```